data/tlg0018/tlg001/tlg0018.tlg001.1st1K-eng1.xml

```diff
@@ -0,0 +1,1013 @@
+<pb n="2"/>
+our own, but instead of much, just a little, such as it may be probable that human intellect may attain to, when wholly occupied with a love of and desire for wisdom.
+</p>
+</div>
+
+<div type="textpart" subtype="section" n="6">
+<p>For as the smallest seal receives imitations of <choice><sic>things or</sic><corr>things of</corr></choice> colossal magnitude when engraved upon it, so perchance in some instances the exceeding beauty of the description of the creation of the world as recorded in the Law, overshadowing with its brilliancy the souls of those who happen to meet with it, will be delivered to a more concise record after these facts have been first premised which it would be improper to pass over in silence.
+<milestone unit="chapter" n="2"/></p>
+</div>
+
+<div type="textpart" subtype="section" n="7">
+<p>For some men, admiring the world itself rather than the Creator of the world, have represented it as existing without any maker, and eternal; and as impiously as falsely have represented God as existing in a state of complete inactivity, while it would have been right on the other hand to marvel at the might of God as the creator and father of all, and to admire the world in a degree not exceeding the bounds of moderation.
+</p>
+</div>
+
+<div type="textpart" subtype="section" n="8">
+<p>But Moses, who had early reached the very summits of philosophy, <ref n="2.1"/> and who had learnt from the oracles of God the most numerous and important of the principles of nature, was well aware that it is indispensable that in all existing things there must be an active cause, and a passive subject; and that the active cause is the intellect of the universe, thoroughly unadulterated and thoroughly unmixed, superior to virtue and superior to science, superior even to abstract good or abstract beauty;</p>
+</div>
+
+<div type="textpart" subtype="section" n="9">
+<p>while the passive subject is something inanimate and incapable of motion by any intrinsic power of its own, but having been set in motion, and fashioned, and endowed with life by the intellect, became transformed into that most perfect work, this world. And those who describe it as being uncreated, do, without being aware of it, cut off the most useful and necessary of all the qualities which tend to produce piety, namely, providence:</p>
+</div>
+
+<div type="textpart" subtype="section" n="10">
+<p>for reason proves that the father and creator has a care for that which has been created; for a father is anxious for the life of his children, and a workman aims at the duration of his works, and employs every device imaginable to
+<note n="2.1">This is in accordance with the description of him in the Bible, where he is represented as being learned in all the wisdom of the Egyptians. </note>
```

Comment:
I believe that the way this is tagged, this note will appear here, inline, nowhere near the thing it is referencing. Simplest thing to do is move it to where it belongs, and drop the n attribute.